clean/ipm_new_orleans_iapro_pprr.py

```diff
@@ -163,12 +170,15 @@ def clean():
         .pipe(clean_races, ["race"])
         .pipe(clean_department_desc)
         .pipe(assign_agency)
-        .pipe(gen_uid, ["agency", "employee_id"])
+        .pipe(
+            gen_uid, ["agency", "employee_id", "first_name", "last_name", "middle_name"]
+        )
```

Comment:
Hmm, is the `employee_id` not unique enough for this dataset?

Reply:
I think that it is possible for two unique persons to have the same `employee_id` at two different points in time.